sklearn/preprocessing/tests/test_data.py

```diff
@@ -227,6 +245,40 @@ def test_min_max_scaler_iris():
     assert_raises(ValueError, scaler.fit, X)
 
 
+def test_min_max_scaler_grouped_iris():
+    # Verify that the `per_feature=False` setting works in the MinMaxScaler.
+    X = iris.data
+    scaler = MinMaxScaler(per_feature=False)
+    # default params
+    X_trans = scaler.fit_transform(X)
+    assert_array_almost_equal(X_trans.min(), 0)
+    assert_array_almost_equal(X_trans.max(), 1)
+    assert_false((X_trans.min(axis=0) == 0).all())
+    assert_false((X_trans.max(axis=0) == 1).all())
+    X_trans_inv = scaler.inverse_transform(X_trans)
+    assert_array_almost_equal(X, X_trans_inv)
+
+    # not default params: min=1, max=2
+    scaler = MinMaxScaler(feature_range=(1, 2))
```

Comment:
copy-paste of test_min_max_scaler_iris().
You need to add `per_feature=False` and change the assertions accordingly
:)